main.js

```diff
@@ -64,6 +65,45 @@ function checkStatus() {
             setConnected(result.alive);
         }
     });
+
+    // Check other read only objects
+    // TODO: This should probably be in it's own function
+    var postData = JSON.stringify({
+		'method' : 'getPowerStatus',
+		'params' : [''],
+		'id' : 1,
+		'version' : '1.0'
+	});
+
+	var options = {
+		host: adapter.config.ip,
+		port: '80',
+		path: '/sony/system',
+		method: 'POST',
+		headers: {
+			'Content-Type': 'application/x-www-form-urlencoded',
+			'Content-Length': postData.length
+		}
+	};
+            
+	var postReq = http.request(options, function(postRes){
+		var body = '';
+		postRes.on('data', function(data){
```

Comment:
Please use arrow functions whenever possible, except when there is a problem with `this` context, which is not the case in the whole code.

Reply:
Look at line 60 of this file. ping.probe line. I follow the same convention. What's the problem?

Reply:
I guess, the code has been written before ES6. So it would be fine to change the old cb functions to arrow functions + adjust yours. But of course it's not a big problem, just like the other changes - more or less optimizations.